src/lock/session.go

```diff
@@ -121,10 +134,10 @@ func (r *Runner) Run() error {
 }
 
 func (r *Runner) startSession(ctx context.Context, db Database) (sessionID int64, err error) {
-	span, spanCtx := r.tracer.StartSpanWithContext(ctx, "runner start session")
+	span, spanCtx := r.client.StartSpanWithContext(ctx, "runner start session")
 	defer func() {
 		if err != nil {
-			span.SetError(err)
+			span.SetTag("error", err)
```

Comment:
I think this is the correct convention for setting error tags on spans: https://github.com/promoboxx/go-service/blob/master/alice/middleware/trace/trace.go#L73